cartographer_rviz/cartographer_rviz/drawable_submap.cc

```diff
@@ -266,15 +273,7 @@ float DrawableSubmap::UpdateAlpha(const float target_alpha) {
 }
 
 void DrawableSubmap::ToggleVisibility() {
-  if (visibility_->getBool()) {
-    if (scene_node_->numAttachedObjects() == 0) {
-      scene_node_->attachObject(manual_object_);
-    }
-  } else {
-    if (scene_node_->numAttachedObjects() > 0) {
-      scene_node_->detachObject(manual_object_);
-    }
-  }
+  scene_node_->setVisible(visibility_->getBool());
```

Comment:
Why this change? I vaguely remember you had the longer code before for some performance reasons or so?